app/views/admin/enterprises/form/_address.html.haml

```diff
@@ -41,8 +41,7 @@
     \/
     = af.label :longitude, t(:longitude)
     %span.required *
-    %div{'ofn-with-tip' => t('latitude_longitude_tip')}
-      %a= t('admin.whats_this')
+    = render partial: 'admin/shared/tooltip', locals: {tooltip_text: t('latitude_longitude_tip')}
```

Comment:
I'm wondering if this one is related to the commit? Not sure to understand ...

Reply:
It should have been in a separate commit, yes, but it's part of the pull request.

Reply:
it was just a small change which is why I didn't make it a separate commit